pkg/autoscaler/scaling/autoscaler_test.go

```diff
@@ -435,6 +445,20 @@ func TestAutoscalerRateLimitScaleUp(t *testing.T) {
 	expectScale(t, a, time.Now(), ScaleResult{100, expectedEBC(10, 61, 1001, 10), na, true})
 }
 
+func TestAutoscalerScaleDownNoLimitWhenNonReachable(t *testing.T) {
+	metrics := &fake.MetricClient{StableConcurrency: 1, PanicConcurrency: 1}
```

Comment:
Where is `Reachable` set to false in this test case?

Reply:
hell, I thought I removed that test. I just went with table tests.

Reply:
removed.